src/hooks/staking/useTotalUnbondingAmount.ts

```diff
@@ -0,0 +1,52 @@
+import { useActiveAccountAddress } from '@recoil/activeAccount';
+import { useQuery } from '@apollo/client';
+import React from 'react';
+import { coin } from '@cosmjs/amino';
+import { useCurrentChainInfo } from '@recoil/settings';
+import { Coin } from '@desmoslabs/desmjs-types/cosmos/base/v1beta1/coin';
+import GetAccountUnbondingDelegationTotal from 'services/graphql/queries/GetAccountUnbondingDelegationTotal';
+
+/**
+ * Hook that provides the total amount of coins that an account is unbonding.
+ * @param userAddress - Address of the account for which the amount will be calculated.
+ * If undefined, the address of the current active user will be used instead.
+ */
+const useTotalDelegatedAmount = (userAddress?: string) => {
+  const activeAccountAddress = useActiveAccountAddress();
+  const address = userAddress ?? activeAccountAddress;
+  const chainInfo = useCurrentChainInfo();
+
+  if (address === undefined) {
+    throw new Error("can't get staked amount without an active account or a provided address");
+  }
+
+  const { data, loading, error, refetch } = useQuery(GetAccountUnbondingDelegationTotal, {
+    variables: {
+      address,
+    },
+    // Use network-only otherwise if the user returns to a screen where this hook
+    // has been used after performing an unbonding action the total amount
+    // will be different from the amount staked on chain.
```

Comment:
```suggestion
    // Use network-only in order to avoid any inconsistency with on-chain data.
    // This might happen if the user returns to a screen where this hook
    // has been used after performing an unbonding action. In this case the total
    // amount shown will be different from the amount staked on chain.
```